services/minespace-web/common/actionCreators/reportActionCreator.js

```diff
@@ -48,11 +48,17 @@ export const createMineReport = (mineGuid, payload) => (dispatch) => {
     });
 };
 
-export const fetchMineReports = (mineGuid) => (dispatch) => {
+export const fetchMineReports = (
+  mineGuid,
+  reportsType = Strings.MINE_REPORTS_TYPE.codeRequiredReports
+) => (dispatch) => {
   dispatch(request(reducerTypes.GET_MINE_REPORTS));
   dispatch(showLoading());
   return CustomAxios()
-    .get(`${ENVIRONMENT.apiUrl}${API.MINE_REPORTS(mineGuid)}`, createRequestHeader())
+    .get(
+      `${ENVIRONMENT.apiUrl}${API.MINE_REPORTS(mineGuid)}?mine_reports_type=${reportsType}`,
+      createRequestHeader()
```

Comment:
Can you move this into the MINE_REPORTS const as another param?
API.MINE_REPORTS(mineGuid, reportType)

Reply:
done